test/mitmproxy/console/test_pathedit.py

```diff
@@ -47,3 +49,25 @@ def test_completion(self):
         s = "thisisatotallynonexistantpathforsure"
         assert c.complete(s) == s
         assert c.final == s
+
+
+class TestPathEdit():
+
+    def test_keypress(self):
+
+        pe = pathedit.PathEdit()
```

Comment:
Actually, I couldn't get the `@patch()` decorators to work. If you'd notice, i create a `PathEdit` object *before* doing the patching or else weird errors were showing up. :confused: